src/partials/guide/style.scss

```diff
@@ -129,15 +129,14 @@ nav {
 
     &:after {
       content: '';
-      background-image: url(templates/guide/images/nav-selected-indicator-mask.svg);
+      background-image: url('partials/guide/images/nav-selected-indicator-mask.svg');
```

Comment:
I think it's mostly a personal preference. There might be cases where not quoting the URL you might run into issues, but I'm not honestly sure. I think maybe in really old browsers...

http://stackoverflow.com/questions/2168855/is-quoting-the-value-of-url-really-necessary